stellar/deploy-contract.js

```diff
@@ -51,24 +54,25 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
     }
 }
 
-async function processCommand(options, config, chain) {
-    const { wasmPath, contractName, privateKey } = options;
+async function deploy(options, config, chain, ...args) {
```

Comment:
```suggestion
async function deploy(options, config, chain, contractName) {
```
Make it positional here as well

Reply:
updated